specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json

```diff
@@ -0,0 +1,2921 @@
+                  "$ref": "./examples/SuccessfulAnalyzeTextJobStatusRequest.json"
+                }
+              },
+              "deprecated": false
+            },
+            "delete": {
+                "produces": [
+                  "application/json",
+                  "text/json"
+                ],
+                "description": "Cancel analysis job.",
+                "operationId": "AnalyzeText_CancelJob",
+                "summary": "Cancel analysis job",
+                "parameters": [
+                  {
+                    "$ref": "#/parameters/JobId"
+                  }
+                ],
+                "responses": {
+                  "202": {
+                    "description": "Cancel Job request has been received.",
+                    "headers": {
+                      "Operation-Location": {
+                        "type": "string"
+                      }
+                    }
+                  },
+                  "default": {
+                    "description": "Unexpected error",
+                    "schema": {
+                      "$ref": "#/definitions/ErrorResponse"
+                    }
+                  }
+                },
+                "x-ms-examples": {
+                  "Successful Delete Text Analysis Job Request": {
+                    "$ref": "./examples/SuccessfulAnalyzeTextJobDeleteRequest.json"
+                  }
+                },
+                "x-ms-long-running-operation": true
+              }
+          }
+    },
+    "definitions": {
+        "AnalyzeTextTaskKind": {
+            "type": "string",
+            "description": "Enumeration of supported Text Analysis tasks.",
+            "enum": [
+                "SentimentAnalysis",
+                "EntityRecognition",
+                "PiiEntityRecognition",
+                "KeyPhraseExtraction",
+                "EntityLinking",
+                "Healthcare",
+                "ExtractiveSummarization",
+                "CustomEntityRecogntion",
+                "CustomSingleClassification",
+                "CustomMultiClassification"
+            ],
+            "x-ms-enum": {
+                "modelAsString": true
+            }
+        },
+        "AnalyzeTextInput": {
+            "allOf": [
+                {
+                    "$ref": "#/definitions/MultiLanguageBatchInput"
+                },
+                {
+                    "$ref": "#/definitions/AnalyzeTextTask"
+                }
+            ]
+        },
+        "AnalyzeTextTask": {
+            "description": "TODO",
+            "discriminator": "task",
+            "required": [
+                "task"
+            ],
+            "properties": {
+                "task": {
+                    "$ref": "#/definitions/AnalyzeTextTaskKind"
+                }
+            }
+        },
+        "AnalyzeTextResponse": {
+            "type": "object",
+            "discriminator": "task",
```

Comment:
It is usually hard to avoid breaking changes on polymorphic types, especially for SDK.

https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#polymorphic-types

Reply:
Can you elaborate more on this?  I recall asking the group during the API review if this would cause any serious problems with the SDK and got very little pushback from the group during the discussion.  I had suspected that this could be a source of issues but due to the level of support received during the review this was the route that we chose to try.